service/grpc/codelingo/codelingo.pb.go

```diff
@@ -370,13 +230,6 @@ func (m *SessionReply) String() string            { return proto.CompactTextStri
 func (*SessionReply) ProtoMessage()               {}
 func (*SessionReply) Descriptor() ([]byte, []int) { return fileDescriptor0, []int{13} }
 
```

Comment:
comment on exported type QueryRequest should be of the form "QueryRequest ..." (with optional leading article)